js/tools/browserManager.js

```diff
@@ -63,7 +63,9 @@ class BrowserManager { /* exported BrowserManager*/
     let dfTab = null;
     let openNewTab = this._alwaysOpenNewTab || openNewTabForce;
     let openNewTabForeground = openNewTabBackGroundForce ? false : this._openNewTabForeground;
-    let reuseDropFeedsTab = this._reuseDropFeedsTab;
+    let feedUrlStartPattern = 'blob:' + browser.extension.getURL('');
+    let isFeed = url.startsWith(feedUrlStartPattern);
```

Comment:
@dauphine-dev - we should probably rework BrowserManager.isDropFeedsTab() to work on URL strings instead duplicating the code here.

Reply:
yeah you are right

Reply:
Done on 0.8 branch